Ontime.js

```diff
@@ -138,13 +143,13 @@ function wsMessageReceived(message) {
   } else if (type == 'ontime-runtime') {
     var runtime = local.values.runtime;
 
-    runtime.selectedEventIndex.set(payload.selectedEventIndex === null ? 0 : payload.selectedEventIndex + 1); // Off by 1, starts at 0, -1 when inactive
+    runtime.activeEventIndex.set(payload.selectedEventIndex === null ? 0 : payload.selectedEventIndex + 1); // Off by 1, 0 when inactive
```

Comment:
Could we use the wording loaded event to someway match Ontime naming

Reply:
The idea was that Active does not imply loaded or playing, does that make sense?

Reply:
Hmm maybe 
But if you load an event in the UI that will become the selectedIndex

And I you play an event it will first be loaded and the played

Reply:
How about 'Current event index'?
It matches the naming already communicated to the user (Current event) so that would hopefully be very clear.